src/main/scala/no/ndla/articleapi/service/ConverterService.scala

```diff
@@ -154,6 +157,7 @@ trait ConverterService {
     def toApiArticle(article: domain.Article): api.Article = {
       api.Article(
         article.id.get.toString,
+        article.revision.get,
```

Comment:
Tja, tjo. I teorien er det jo slik. Bruken av `toApiArticle` tilsier at det alltid er en persistert artikkel som returneres (som vi ved hjelp av database constraints) som vi vet har en `revision` og en `id`, så dersom `revision` eller `id` er undefined så vil det ha feilet før denne metoden benyttes..

Så om vi får det til uten at det forkludrer for mye av koden så er jeg for å slippe `get`.
Uansett, dersom `revision` eller `id` er None, så er det eneste vi kan gjøre her å kaste en feil.